Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -774,6 +791,151 @@ def validate_no_multiple_indicators_for_search(arg_list):
+                'Description': tag.get('description', '')})
+        indicator['Tags'] = tags
+
+    if indicator_type == 'Domain':
+        indicator['WhoisAdminCountry'] = res.get('whoisAdminCountry', '')
+        indicator['WhoisAdminEmail'] = res.get('whoisAdminEmail', '')
+        indicator['WhoisAdminName'] = res.get('whoisAdminName', '')
+        indicator['WhoisDomainCreationDate'] = res.get('whoisDomainCreationDate', '')
+        indicator['WhoisDomainExpireDate'] = res.get('whoisDomainExpireDate', '')
+        indicator['WhoisDomainUpdateDate'] = res.get('whoisDomainUpdateDate', '')
+        indicator['WhoisRegistrar'] = res.get('whoisRegistrar', '')
+        indicator['WhoisRegistrarUrl'] = res.get('whoisRegistrarUrl', '')
+        indicator['WhoisRegistrant'] = res.get('whoisRegistrant', '')
+
+    return indicator
+
+
+def calculate_dbot_score(indicator_response, indicator_type):
+    latest_pan_verdicts = indicator_response['indicator']['latestPanVerdicts']
+    if not latest_pan_verdicts:
+        raise Exception('latestPanVerdicts value is empty in indicator response.')
+
+    pan_db = latest_pan_verdicts.get('PAN_DB')
+    wf_sample = latest_pan_verdicts.get('WF_SAMPLE')
+
+    # use WF_SAMPLE value for file indicator and PAN_DB for domain,url and ip indicators
+    if indicator_type == 'File' and wf_sample:
+        return VERDICTS_TO_DBOTSCORE.get(wf_sample.lower(), 0)
+    elif pan_db:
+        return VERDICTS_TO_DBOTSCORE.get(pan_db.lower(), 0)
+    else:
+        score = next(iter(latest_pan_verdicts.values()))
+        return VERDICTS_TO_DBOTSCORE.get(score.lower(), 0)
+
+
+def get_indicator_outputs(indicator_type, indicator_response, indicator_value, score, indicator_context_output):
+    dbot_score = {
+        'Indicator': indicator_value,
+        'Type': indicator_type.lower(),
+        'Vendor': VENDOR_NAME,
+        'Score': score
+    }
+
+    indicator_context = {
+        indicator_context_output: indicator_value
+    }
+
+    if score == 3:
+        indicator_context['Malicious'] = {
+            'Vendor': VENDOR_NAME
+        }
+
+    if indicator_type == 'Domain':
+        whois = dict()  # type: ignore
+        whois['Admin'] = dict()
+        whois['Registrant'] = dict()
+        whois['Registrar'] = dict()
+        whois['CreationDate'] = indicator_response['WhoisDomainCreationDate']
+        whois['ExpirationDate'] = indicator_response['WhoisDomainExpireDate']
+        whois['UpdatedDate'] = indicator_response['WhoisDomainUpdateDate']
+        whois['Admin']['Email'] = indicator_response['WhoisAdminEmail']
+        whois['Admin']['Name'] = indicator_response['WhoisAdminName']
+        whois['Registrar']['Name'] = indicator_response['WhoisRegistrar']
+        whois['Registrant']['Name'] = indicator_response['WhoisRegistrant']
+        indicator_context['WHOIS'] = whois
+
+    ec = {f'AutoFocus.{indicator_type}': indicator_response, 'DBotScore': dbot_score, indicator_type: indicator_context}
```

Comment:
context path is not `indicator_type`, where is the DT?
use `outputPaths` dictionary from `CommonServerPython`